src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md

```diff
@@ -308,6 +308,16 @@ Successful: 1993 files; errors: 0
 ---
 ```
 
+The following command deploy only js with 4 jobs, deployment strategy is standard
```

Comment:
```suggestion
The following command deploys only JS with 4 jobs, deployment strategy is standard
```

Reply:
Hello @BarnyShergold 
Thank you for the suggestion, I have update PR, can you review it?